contrib/devtools/prepare_release/prepare_release_config.yaml

```diff
@@ -0,0 +1,79 @@
+# THIS IS A TEMPLATE FILE, FOR EXAMPLES ON ACTUAL RELEASES CHECK CONTENT OF prepare_release-X.Y.Z.yaml FILES
+# path to zend project root
+repository_root: "./"
+# new release version (format "major.minor.patch")
+version: ""
+# build number
+build: ""
+# mainnet config
+mainnet:
+  # height for checkpoint (if empty, current last block is used)
+  checkpoint_height: ""
+  # total transaction from genesis block up to checkpoint (can be found in "debug.log" file)
+  total_transactions: ""
+# testnet config
+testnet:
+  # height for checkpoint (if empty, current last block is used)
+  checkpoint_height: ""
+  # total transaction from genesis block up to checkpoint (can be found in "debug.log" file)
+  total_transactions: ""
+# release date (format "%a, %d %b %Y")
+release_date: ""
```

Comment:
Can we add an example date to make it more immediate to understand the format string?

Reply:
In some sense I agree with you, but I prefer to keep this file as a template (no examples included); we agreed on inserting the config file used for each release generation in the `zen/contrib/devtools/prepare_release` folder (similar to what we already do for release notes files), so if a user need to check for an example there are actual files available for inspection.

Reply:
Updated just the comment.